openboxes/openboxes
# OBPIH-6397 Full outbound import - fix QA issues

QA issues fixed:
 - make the recipient nullable and change the error message if the recipient was provided, but not found
-  persist tracking number to make it visible on SM view page
- unify ship date on create full outbound page and on outbound send page (time context)
- validate the empty packing list (do not allow to import an empty template)
- validate quantity picked 0.
- make the expiration date not bindable, and display it in the table after associating the inventory item with the provided lot number
- improve display of validation errors on frontend (a case when we are on confirm page and for example recall a lot on second tab and then run finish)
- unify the name of date shipped field
- fix issue with "empty" transaction entries being persisted, that then caused the transaction view page not working
